packages/service-client/odsp-client/package.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
Let's be sure to add a README in this package explaining what it does, how to use it, etc. (preferably with examples).